test/aircraft/FlightManagementSystem/RouteModel.spec.js

```diff
@@ -953,6 +953,16 @@ ava('._createAmendedConvergentLeg() returns an empty array when leg is a direct
     t.deepEqual(result, []);
 });
 
+ava('._createAmendedConvergentLeg() returns the leg unmodified when convergent waypoint is the SID leg\'s first fix', (t) => {
+    const model = new RouteModel('KLAS07R.BOACH6.HEC');
+    const waypointName = 'JESJI';
+    const legIndex = 0;
+    const expectedResult = [model._legCollection[legIndex]];
```

Comment:
You should specify the expected result inline here instead of relying on the model to produce it for you.  Doing it this way can lead to hard-to-track side affects and also makes the test easier to read, you can see exactly what is expected without having to inspect the result.

Reply:
Changed to explicitly specify the desired waypoints.

done